### DISCUSSION TITLE
.vitepress folder is ignored?

### DISCUSSION THREAD

**Discussion by antonreshetov** [github-linguist/linguist#6026] [Classification] [answered]
All in `.vitepress` folder not detected as languages. According to linguist the repo contains only JavaScript and Shell, but this is not true, the main files are in the `/docs/.vitepress` folder.

Example - https://github.com/massCodeIO/masscode.io.

**Comment by lildude** [comment, marked as answer]
> All in `.vitepress` folder not detected as languages.

It's not the `.vitpress` folder that is being excluded; it's everything under its parent directory `/docs` as Linguist considers this folder as documentation:

https://github.com/github/linguist/blob/1f65799270b46a0bd1d4a62ea5734a1117f45e61/lib/linguist/documentation.yml#L1-L12

... and thus does not include the content when performing the analysis by default.

If you want everything under `/docs` to be included, you'll need to implement an [override](https://github.com/github/linguist/blob/master/docs/overrides.md#documentation).

**Reply by antonreshetov** [reply]
Thanks
